From 806f98e97aa39fac55787f4eaf916589614fea12 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Wed, 18 Dec 2019 15:27:33 -0800
Subject: [PATCH 606/830] ANDROID: gki_defconfig: Enable
 CONFIG_GNSS_CMDLINE_SERIAL
X-NVConfidentiality: public

Enables a GNSS serial driver for cuttlefish.

Bug: 146517987
Change-Id: Ife42306a3052e49365e3f0004f220424f0be7782
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index d105c67..1bd5593 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -216,6 +216,7 @@ CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
 # CONFIG_ALLOW_DEV_COREDUMP is not set
 CONFIG_DEBUG_DEVRES=y
 CONFIG_GNSS=y
+CONFIG_GNSS_CMDLINE_SERIAL=m
 CONFIG_ZRAM=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_RAM=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 1e10cf9..d3ce2fc 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -189,6 +189,7 @@ CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
 # CONFIG_ALLOW_DEV_COREDUMP is not set
 CONFIG_DEBUG_DEVRES=y
 CONFIG_GNSS=y
+CONFIG_GNSS_CMDLINE_SERIAL=m
 CONFIG_OF=y
 CONFIG_ZRAM=y
 CONFIG_BLK_DEV_LOOP=y
-- 
2.7.4

